docs/source/deployment/databricks.md

```diff
@@ -223,3 +223,68 @@ Out[12]: {}
 Your complete notebook should look similar to this (the results are hidden):
 
 ![](../meta/images/databricks_notebook_example.png)
+
+
+### 9. Using IPython Extension 
+
+You can interact with Kedro in Databricks through the Kedro [IPython extension](https://ipython.readthedocs.io/en/stable/config/extensions/index.html), `kedro.extras.extensions.ipython`.
+
+The Kedro IPython extension launches a [Kedro session](../kedro_project_setup/session.md) and makes available the useful Kedro variables `catalog`, `context`, `pipelines` and `session`. It also provides the `%reload_kedro`  [line magic](https://ipython.readthedocs.io/en/stable/interactive/magics.html) that reloads these variables (for example, if you need to update `catalog` following changes to your Data Catalog).
+
+The IPython extension can be used in a Databricks notebook in a similar way to how it is used in [Jupyter notebooks](https://kedro.readthedocs.io/en/0.18.0/tools_integration/ipython.html)
+
+There's one additional installation you need to do in the Databricks notebook to make use of the IPython extension. After you load the IPython extension using the below command:
+
+```ipython
+In [1]: %load_ext kedro.extras.extensions.ipython
+```
+
+you must explicitly upgrade your pip version by doing the below:
+
+```bash
+%pip install -U pip
+```
+
+after this, you can reload kedro by running the line magic command  `%reload_kedro <path_to_project_root>`.
+
+### 10. Running Kedro-viz on Databricks 
+
+Currently, if you try to run `%run_viz` on Databricks it will only display the below: 
+
+```console
+<IPython.core.display.HTML object>
+```
+
+While we fix this issue, we have a temporary workaround to run Kedro-viz on Databricks. 
+
+To run Kedro-viz, first run the below command in your Databricks notebook: 
+
+```bash
+%sh kedro viz --no-browser --host 0.0.0.0 --port 4141
+```
+
+```eval_rst
+.. note:: The command execution continues to run and will need to be cancelled manually before proceeding to the next step. Cancelling the command will not quit the kedro-viz server.
+```
+
+After this, you must try and run an example Shiny app using the below command: 
+
+```bash
+%r
+library(shiny)
+runExample("01_hello")
+```
+
+![](../meta/images/databricks_shiny_command.png)
+
+When you click on the Shiny app link, it will open a browser with an example Shiny app running. Now edit the port at the end of the URL and change it to 4141. 
+
+![](../meta/images/databricks_kedro_viz.gif)
+
+You will notice Kedro-viz is blank at first, but if you click on the flowchart tab, Kedro-viz will run as normal. 
+
+```eval_rst
+.. note:: Ctrl+C to quit the Kedro-viz server doesn't work on Databricks. To end the process you will need to find the process-id on port=4141 or the port you used and kill it using the Linux command kill -9 PID
```

Comment:
```suggestion
.. note:: Ctrl+C to quit the Kedro-viz server doesn't work on Databricks. To end the process you will need to find the process-id on port=4141 or the port you used and kill it using the Linux command `kill -9 PID`.
```

Reply:
I don't know if my command highlight is correct here.

Reply:
Is Ctrl+C what users would do both on Mac on Windows?

Reply:
yes!